dbms/src/AggregateFunctions/AggregateFunctionMinMaxWindow.h

```diff
@@ -251,12 +220,12 @@ struct AggregateFunctionMaxDataForWindow : Data
 
     void changeIfBetter(const IColumn & column, size_t row_num, Arena * arena)
     {
-        return this->add(column, row_num, arena);
+        return this->template add<false>(column, row_num, arena);
```

Comment:
Is it a bug here?

Reply:
> Is it a bug here?

No, just compilation error after changed